casper/src/main/scala/coop/rchain/casper/CasperConf.scala

```diff
@@ -34,5 +34,6 @@ final case class GenesisBlockData(
     genesisBlockNumber: Long,
     numberOfActiveValidators: Int,
     posMultiSigPublicKeys: List[String],
+    posVaultPubKey: String,
```

Comment:
Keep `posMultiSigPublicKeys` and `posMultiSigQuorum` parameters together. They are part of the same functionality.